GNUmakefile

```diff
@@ -0,0 +1,76 @@
+# User-customizable variables:
```

Comment:
Why is this `GNUmakefile` rather than just `Makefile`? `Makefile` is much more common and GNU make actually does not recommend using `GNUmakefile`:
https://www.gnu.org/software/make/manual/html_node/Makefile-Names.html

Reply:
Are you referring to the following sentence:

> The first name checked, `GNUmakefile`, is not recommended for most makefiles.

Notice this sentence immediately after:

> You should use [`GNUmakefile`] if you have a makefile that is specific to GNU `make`, and will not be understood by other versions of `make`.

If this file was using only standard `make` features, I would name it `Makefile`. However, this file uses a few GNU-specific features, so I wanted to be clear that this makefile only works with GNU `make`.

I can definitely rename it to `Makefile` if you prefer that name.

Reply:
Yes, I would prefer this to be renamed to Makefile. This is the first time it see GNUmakefile in my entire life.